### PR TITLE
Feat/profile form UI fixes

### PR DESCRIPTION
**Pull request by AniaPiWo**

### CODE REVIEW

**Review by hansac91 — APPROVED** [review]

**Review by wawrzonnn — APPROVED** [review]
FB_IMG_1700672571039

File: app/(profiles)/my-profile/create/loading.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import styles from './page.module.scss'
+
+const Loading = () => {
```
Comment (Sumick) [inline app/(profiles)/my-profile/create/loading.tsx:4]: **nitpick:** I think we should reuse this loader, not duplicate it

**Review by Sumick — CHANGES_REQUESTED** [review]
We should fix this Loader to not duplicate it every time - it should be some one component, and then reused

**Review by Sumick — APPROVED** [review]
🔥🔥🔥

**Pull request merged**